ui/app/components/clients/dashboard.js

```diff
@@ -75,135 +75,20 @@ export default class Dashboard extends Component {
     return this.args.model.activity && this.args.model.activity.total;
   }
 
-  // Construct the namespace model for the search select component
-  get searchDataset() {
-    if (!this.args.model.activity || !this.args.model.activity.byNamespace) {
+  // top level TOTAL client counts from response for given date range
+  get runningTotals() {
+    if (!this.args.model.newInitActivity || !this.args.model.newInitActivity.total) {
```

Comment:
depending on when you merge, this will become this.args.activity (I've replaced all newInitActivity with activity